hwpf/fapi2/src/plat/plat_error.C

```diff
@@ -0,0 +1,56 @@
+#include "plat_error.H"
+#include "plat_utils.H"
+
+namespace fapi2
+{
+/**
+ * @brief Used to special handling of clock errors.
+ *
+ * The defined api is used to modify the callout and guard info
+ * specific to clock failures.
+ *
+ * @param[in,out] ffdc        ffdc data which need to be updated if it is clock failure
+ * @param[in,out] hwCallout   reference of hwcallout with clock related hwid
+ * @param[in] guardRefTarget  indicates whether reference target need to be guarded or not
+ *
+ * @return void
+ */
+static void process_clock_callout(FFDC &ffdc, HWCallout& hwCallout, const bool guardRefTarget)
+{
+	// Planar callout is required since, clock is not a FRU, its parent has
+	// to be called out, and clock targets parent is Planar.
+	hwCallout.isPlanarCallout = true;
+
+	// This is an override to CDG defined in error xml. We cant add this guard
+	// in xml, since xml is comon for other systems also, where guard is not
```

Comment:
typo: comon -> common
> other systems

I guess you meant it as other service processor platform systems